doc/source/tune/faq.rst

```diff
@@ -634,7 +634,7 @@ You can configure this by setting the `RAY_CHDIR_TO_TRIAL_DIR=0` environment var
 This explicitly tells Tune to not change the working directory
 to the trial directory, giving access to paths relative to the original working directory.
 One caveat is that the working directory is now shared between workers, so the
-:meth:`train.get_context().get_trial_dir() <ray.train.context.TrainContext.get_.get_trial_dir>`
+:meth:`ray.train.context.TrainContext.get_trial_dir`
```

Comment:
Would we be able to actually use `train.get_context().get_trial_dir()` as the text? Since it may not be obvious how to retrieve the `TrainContext`.

Reply:
Sure, I'll do that now.